### PR TITLE
python3Packages.pyefergy: init at 0.1.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
An API library for Efergy energy meters

https://github.com/tkdrob/pyefergy

This is a Home Assistant dependency.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 140274` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python38Packages.iso4217</li>
    <li>python38Packages.pyefergy</li>
    <li>python39Packages.iso4217</li>
    <li>python39Packages.pyefergy</li>
  </ul>
</details>

File: pkgs/development/python-modules/iso4217/default.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  preBuild = ''
+    # The table is already downloaded
+    export ISO4217_DOWNLOAD=0
+    # Copy the table file to satifiy the build process
+    cp -r ${table} /build/source/$pname/table.xml
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/iso4217/default.nix:34]: Please don't reference /build. Can we use a relative path?

Comment (fabaff) [reply pkgs/development/python-modules/iso4217/default.nix]: Yes, relative path would work.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 140274` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
4 packages built:
</summary>
<ul>
<li>python38Packages.iso4217</li>
<li>python38Packages.pyefergy</li>
<li>python39Packages.iso4217</li>
<li>python39Packages.pyefergy</li>
</ul>
</details>

**Pull request merged**